azure-pipelines.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
         azdev --version
 
-        python -m pytest azdev/ --ignore=azdev/mod_templates --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html
+        python -m pytest azdev/ --ignore=azdev/mod_templates --junitxml=junit/test-results.xml --cov=azdev --cov-report=xml --cov-report=html
```

Comment:
Is this a test coverage for azdev?

Reply:
Yeah. For azdev only. Have to use --cov=azdev to indicate the desired pacakage/module to run with it. Otherwise, the other code would get chance to analyse too as the azure-cli and azure-cli-extension is in the placed side by side with azdev/